test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -286,4 +286,19 @@ def test_successful_validate_pinless_debit_card
     assert_equal 'Y', response.params["status"]
     assert_equal true,  response.success?
   end
+
+  def test_network_tokenization_authorize_and_capture
+    credit_card = network_tokenization_credit_card('4111111111111111',
+      :brand              => 'visa',
+      :eci                => "05",
+      :payment_cryptogram => "EHuWW9PiBkWvqE5juRwDzAUFBAk="
```

Comment:
this is from the cybersource docs. there's no way to generate this without initiating an actual apple pay payment